library/pk_wrap.c

```diff
@@ -78,9 +78,79 @@ static size_t rsa_get_bitlen( const void *ctx )
     return( 8 * mbedtls_rsa_get_len( rsa ) );
 }
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
 static int rsa_verify_wrap( void *ctx, mbedtls_md_type_t md_alg,
                    const unsigned char *hash, size_t hash_len,
                    const unsigned char *sig, size_t sig_len )
+{
+    mbedtls_rsa_context * rsa = (mbedtls_rsa_context *) ctx;
+    int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    mbedtls_svc_key_id_t key_id = MBEDTLS_SVC_KEY_ID_INIT;
+    psa_status_t status;
+    mbedtls_pk_context key;
+    int key_len;
+    unsigned char buf[MBEDTLS_PK_RSA_PUR_DER_MAX_BYTES];
+    mbedtls_pk_info_t pk_info = mbedtls_rsa_info;
+    psa_algorithm_t psa_alg_md = PSA_ALG_RSA_PKCS1V15_SIGN( mbedtls_psa_translate_md( md_alg ) );
```

Comment:
There's a hard limit of 80 characters, that's usually only violated by comments, or some macros.

Reply:
Actually we really prefer comments to stick to 80 columns. The cases where we're ok with going beyond 80 columns are mainly:
• Avoid breaking string literals, so that you can easily grep error messages in the source code.
• It's ok to let “boring” stuff go beyond 80 columns sometimes, e.g. `) ) );`
* After refactoring that e.g. gives a symbol a longer name or increases indentation, you don't need to be very strict around rewrapping lines.

Here, please break the line after the `=`.